webpack-dev-server.el

```diff
@@ -62,18 +60,11 @@
   (kill-webpack-dev-server)
   (setq-local default-directory webpack-dev-server-project-root)
   (let ((webpack-dev-server-buf (get-buffer-create webpack-dev-server-buf-name)))
-    (display-buffer
-     webpack-dev-server-buf
-     '((display-buffer-at-bottom
-        display-buffer-pop-up-window
-        display-buffer-reuse-window)
-       (window-height . 30)))
```

Comment:
This part is probably ok. The window height explicitly set was actually used by ghcid but not webpack.